flake.lock

```diff
@@ -0,0 +1,291 @@
+{
+  "nodes": {
+    "common-infra": {
+      "inputs": {
+        "deploy": "deploy",
+        "nixpkgs": "nixpkgs"
+      },
+      "locked": {
+        "lastModified": 1612203367,
+        "narHash": "sha256-z4knCUz+FJxqEgbT+EEFuzpAFWHmWBfAfTr3HuHMKE0=",
+        "owner": "serokell",
+        "repo": "common-infra",
+        "rev": "c5b609969578f8f5ca56c7d439eb010fcbe17157",
+        "type": "github"
+      },
+      "original": {
+        "owner": "serokell",
+        "repo": "common-infra",
+        "type": "github"
+      }
+    },
+    "deploy": {
+      "inputs": {
+        "flake-compat": "flake-compat",
+        "naersk": "naersk",
+        "nixpkgs": [
+          "nixpkgs"
+        ],
+        "utils": "utils"
+      },
+      "locked": {
+        "lastModified": 1608488342,
+        "narHash": "sha256-JhEoKKmETJsdgS+kjT1rACnlM50q/JdVNyAfIbbVQcc=",
+        "owner": "serokell",
+        "repo": "deploy-rs",
+        "rev": "91e934fd9e1e06044f75a3e3ad998813c2a90fb5",
+        "type": "github"
+      },
+      "original": {
+        "owner": "serokell",
+        "repo": "deploy-rs",
+        "type": "github"
+      }
+    },
+    "deploy-rs": {
```

Comment:
I know it's not ready for review yet, but keep in mind that many things here are unused (AFAIU) and should be removed, e. g. I doubt we use `deploy-rs`.